app/helpers/lookup_keys_helper.rb

```diff
@@ -110,38 +111,38 @@ def lookup_key_description(lookup_key, matcher, inherited_value)
   end
 
   def lookup_key_warnings(required, has_value)
-    return { :text => '', :icon => 'info-sign' } if has_value
+    return { :text => '', :icon => 'info-circle' } if has_value
 
     if required
       { :text => _("Required parameter without value.<br/><b>Please override!</b><br/>"),
-        :icon => "warning-sign" }
+        :icon => "exclamation-triangle" }
     else
       { :text => _("Optional parameter without value.<br/><i>Will not be sent to Puppet.</i><br/>"),
-        :icon => "exclamation-sign" }
+        :icon => "exclamation-circle" }
     end
   end
 
   def override_toggle(overridden)
     return unless can_edit_params?
-    link_to_function(icon_text('edit'), "override_class_param(this)",
+    link_to_function(icon_text('pencil-square-o', '', :kind => 'fa'), "override_class_param(this)",
                      :title => _("Override this value"),
                      :'data-tag' => 'override',
                      :class =>"btn btn-default btn-md btn-override #{'hide' if overridden}") +
-      link_to_function(icon_text('remove'), "override_class_param(this)",
+      link_to_function(icon_text('times', '', :kind => 'fa'), "override_class_param(this)",
                        :title => _("Remove this override"),
                       :'data-tag' => 'remove',
                       :class =>"btn btn-default btn-md btn-override #{'hide' unless overridden}")
   end
 
-  def hidden_toggle(hidden, hide_icon = 'eye-close', unhide_icon = 'eye-open', strikethrough = false)
+  def hidden_toggle(hidden, hide_icon = 'eye-slash', unhide_icon = 'eye', strikethrough = false)
     return unless can_edit_params?
     if strikethrough && !hidden
-      link_to_function(icon_text(hide_icon), "", :class =>"btn btn-default btn-md btn-hide disabled")
+      link_to_function(icon_text(hide_icon, '', :kind => 'fa'), "", :class =>"btn btn-default btn-md btn-hide disabled")
     else
-      link_to_function(icon_text(unhide_icon), "input_group_hidden(this)",
+      link_to_function(icon_text(unhide_icon, '', :kind => 'fa'), "input_group_hidden(this)",
```

Comment:
Can we use the same eye-slash icon on Puppet class parameters? The 'A' we're using is less clear.